### PR TITLE
St/evm 480

### PR DESCRIPTION
**Pull request by saratnt**

### CODE REVIEW

File: sdk/src/main/scala/com/horizen/chain/SidechainBlockInfo.scala [hunk 1]
```diff
@@ -44,17 +44,16 @@ object SidechainBlockInfo {
...
-  def mainchainReferenceDataHeaderHashesFromBlock(sidechainBlock: AccountBlock): Seq[MainchainHeaderHash] = {
-    sidechainBlock.mainchainBlockReferencesData.map(data => byteArrayToMainchainHeaderHash(data.headerHash))
   }
+//
+//  def mainchainReferenceDataHeaderHashesFromBlock(sidechainBlock: SidechainBlock): Seq[MainchainHeaderHash] = {
+//    sidechainBlock.mainchainBlockReferencesData.map(data => byteArrayToMainchainHeaderHash(data.headerHash))
+//  }
+//  def mainchainReferenceDataHeaderHashesFromBlock(sidechainBlock: AccountBlock): Seq[MainchainHeaderHash] = {
+//    sidechainBlock.mainchainBlockReferencesData.map(data => byteArrayToMainchainHeaderHash(data.headerHash))
+//  }
```
Comment (i-Alex) [inline sdk/src/main/scala/com/horizen/chain/SidechainBlockInfo.scala:56]: commented code can be removed

File: sdk/src/main/scala/com/horizen/account/AccountSidechainNodeViewHolder.scala [hunk 2]
```diff
@@ -177,99 +175,26 @@ class AccountSidechainNodeViewHolder(sidechainSettings: SidechainSettings,
...
-          val l: AccountNodeView = new AccountNodeView(history(), minimalState(), vault(), memoryPool())
-          sender() ! f(l,functionParams)
-        }
-        catch {
-          case e: Exception => sender() ! akka.actor.Status.Failure(e)
-        }
-
-      }
-  }
+  protected def getNodeView(): AccountNodeView = new AccountNodeView(history(), minimalState(), vault(), memoryPool())
```
Comment (i-Alex) [inline sdk/src/main/scala/com/horizen/account/AccountSidechainNodeViewHolder.scala:178]: add `override`

File: sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala [hunk 3]
```diff
@@ -206,76 +207,7 @@ class SidechainNodeViewHolder(sidechainSettings: SidechainSettings,
...
-        case AbstractSidechainNodeViewHolder.ReceivableMessages.ApplyBiFunctionOnNodeView(f, functionParams) => try {
-          val l: SidechainNodeView = new SidechainNodeView(history(), minimalState(), vault(), memoryPool(), applicationState, applicationWallet)
-          sender() ! f(l, functionParams)
-        }
-        catch {
-          case e: Exception => sender() ! akka.actor.Status.Failure(e)
-        }
-      }
-  }
+  protected def getNodeView(): SidechainNodeView = new SidechainNodeView(history(), minimalState(), vault(), memoryPool(), applicationState, applicationWallet)
```
Comment (i-Alex) [inline sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala:210]: add `override`

**Review by i-Alex — CHANGES_REQUESTED** [review]
looks good!

File: sdk/src/main/scala/com/horizen/chain/SidechainBlockInfo.scala (hunk 1 above)
Comment (saratnt) [reply sdk/src/main/scala/com/horizen/chain/SidechainBlockInfo.scala:56]: OK

**Review by i-Alex — APPROVED** [review]

**Pull request merged**